python_script/meter-health/meter_health.py

```diff
@@ -0,0 +1,119 @@
+import requests
+import csv
+import logging
+logging.basicConfig(filename='example.log',level=logging.DEBUG)
+from oauthlib.oauth2 import LegacyApplicationClient
+from requests_oauthlib import OAuth2Session
+from config.credentials import *
+
+
+class MeterHealth:
+    # URL information for the API. ONLY change this method if you know what you are doing
+    BASE_URL="http://54.90.173.204:8080"
+    ORGANIZATIONS="/api/v2.json"
+    METERS="/api/v2/organizations/%d/meters.json"
+    TOKEN_URL="/oauth/token"
+
+    def __init__(self):
+        self.token=self.retrieve_token()
+        self.payload={'access_token': self.token}
+
+    def create_report(self):
+        """ It is the main method that calls
+        all that need to be done
+        """
+        if self.token:
+            organizations=self.retrieve_organizations()
+            if organizations:
+                self.build_csv(organizations)
+            else:
+                print("There aren't organizations \n")
+
+    def build_csv(self, organizations):
+        # Creates/Opens the file where the info will be created
```

Comment:
You could replace this comment with: `# Creates/opens the file which will store the info`